mealie/routes/households/controller_invitations.py

```diff
@@ -21,18 +21,31 @@
 class GroupInvitationsController(BaseUserController):
     @router.get("", response_model=list[ReadInviteToken])
     def get_invite_tokens(self):
-        return self.repos.group_invite_tokens.page_all(PaginationQuery(page=1, per_page=-1)).items
+        if self.user.admin:
```

Comment:
Can we move this to an admin route? I'm worried it might get confusing if we respond with all tokens sometimes, and tokens filtered by group other times. Maybe make a new admin controller for invitations?

Reply:
Thanks for the feedback! Indeed it seems more appropriate to me in a separate route (see `mealie/routes/admin/admin_invitations.py`). I also added a `get_one()` operation that allows to specify the *group_slug* or the *id*.